org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/JavaEclipseCompiler.scala

```diff
@@ -52,4 +53,7 @@ class JavaEclipseCompiler(p: IProject, monitor: SubMonitor) extends JavaCompiler
       refresh()
     }
   }
+
+  def compileWithReporter(sources: Array[File], classpath: Array[File], output: Output, options: Array[String], reporter: Reporter, log: Logger): Unit =
```

Comment:
Does this method override/implement an interface method? If yes, please mark it with the `override` keyword, otherwise add Scaladoc about its purpose.